lib/src/api/engine/remote/ws/native.rs

```diff
@@ -110,14 +114,14 @@ impl Connection for Client {
 			#[cfg(not(any(feature = "native-tls", feature = "rustls")))]
 			let maybe_connector = None;
 
+			#[allow(deprecated)]
 			let config = WebSocketConfig {
-				max_send_queue: match capacity {
-					0 => None,
-					capacity => Some(capacity),
-				},
 				max_message_size: Some(MAX_MESSAGE_SIZE),
 				max_frame_size: Some(MAX_FRAME_SIZE),
 				accept_unmasked_frames: false,
+				write_buffer_size: Default::default(),
+				max_write_buffer_size: MAX_WRITE_BUFFER_SIZE,
+				max_send_queue: None,
```

Comment:
Same here. Let's please remove this line and the `#[allow(deprecated)]`.

Reply:
We can't remove the line. As this is Rust, removing the line will cause the project to fail to compile. It has to stay until tungstenite officially decides to remove the field.

Reply:
`WebSocketConfig` implements `Default` so we can remove it and add `..Default::default()` at the end:

```rust
let config = WebSocketConfig {
    max_message_size: Some(MAX_MESSAGE_SIZE),
    max_frame_size: Some(MAX_FRAME_SIZE),
    max_write_buffer_size: MAX_WRITE_BUFFER_SIZE,
    ..Default::default()
};
```

Reply:
Ah, I get what you mean. Thaks for the suggestion. I'll go ahead and add that now.

Reply:
Pushed up the change. Let me know if it matches your expectecations.

Reply:
That's perfect. Thank you!